modules/persistent-storage-csi-azure-file-nfs.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-azure-file-nfs_{context}"]
+= NFS support
```

Comment:
```suggestion
= Network File System support
```

Reply:
This is the first time that NFS is mentioned in this assembly so I'd recommend writing it out.